freedommaker/tests/vmdebootstrap-stub

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python3
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
```

Comment:
This is a simple copy/paste error.  I have modified my commit and pushed the newer version. I have also added another commit updating the README.